### PR TITLE
Added 4 bibitems in 2021.bib and 2022.bib

### PR DESCRIPTION
**Pull request by richardschu**
Appended the items to the respective files.

### CODE REVIEW

**Comment by peterrum** [comment]
> ERROR: found non-ASCII characters!

**Comment by richardschu** [comment]
Jenkins tells me the eact line, right?

**Review by peterrum — APPROVED** [review]
Would you mind to squash the commits?

**Comment by richardschu** [comment]
Thanks Peter, have a great weekend ; )

File: publications-2021.bib [hunk 1]
```diff
@@ -693,4 +693,42 @@ @article{Sabanskis2021
...
+}
+
+@article{Schussnig2021,
+  title = {Robust stabilised finite element solvers for generalised {N}ewtonian fluid flows},
+  journal = {Journal of Computational Physics},
+  volume = {442},
+  pages = {110436},
+  year = {2021},
+  issn = {0021-9991},
+  doi = {https://doi.org/10.1016/j.jcp.2021.110436},
```
Comment (marcfehling) [inline publications-2021.bib:729]:
```suggestion
  doi = {10.1016/j.jcp.2021.110436},
```

File: publications-2022.bib [hunk 2]
```diff
@@ -118,4 +118,16 @@ @article{Sabanskis2022
...
 }
 
+@article{Schussnig2022,
+title = {Efficient split-step schemes for fluid--structure interaction involving incompressible generalised {N}ewtonian flows},
+journal = {Computers \& Structures},
+volume = {260},
+pages = {106718},
+year = {2022},
+issn = {0045-7949},
+doi = {https://doi.org/10.1016/j.compstruc.2021.106718},
```
Comment (marcfehling) [inline publications-2022.bib:128]:
```suggestion
doi = {10.1016/j.compstruc.2021.106718},
```

**Review by marcfehling — CHANGES_REQUESTED** [review]

**Comment by richardschu** [comment]
I will change that now and squash again!

**Comment by richardschu** [comment]
I think its good now, is it?

**Review by marcfehling — APPROVED** [review]
Looks good to me!

**Pull request merged**